### ISSUE TITLE
Alert banner focus links have insufficient focus contrast

### ISSUE THREAD

**Issue by danchamp** [localgovdrupal/localgov_base#108]
[Accessibility audit: issues 17-19](https://docs.google.com/spreadsheets/d/1t8rRIQe_1kogxPBZDKcsbf8YxtEfC9iGJvidkagNSXY/edit#gid=1779042480&range=E18)

Affects death of a notable person, announcement, and emergency alert banners.
